### DISCUSSION TITLE
regarding capillary pressure: the van genuchten model

### DISCUSSION THREAD

**Discussion by marinsiebert** [idaholab/moose#26790] [Q&A Modules: Porous Flow]
Hello, 
this is an outsourced discussion from #26788:

A simulation of an unsaturated soil with a heatsource is done to fit experimental data. While the temperatures are fitting very nicely, the capillary pressure is still a bite of a riddle. 
In the porous flow module,  the van genuchten function for "capillary pressure" is used, originating from the equation for saturation as a function of the pressure head - meaning that the capillary pressure would be the same as the matric potential? 

The results of the experiment are given as matric potential (which would be the capillary pressure according to this), however the measured potential is way more sensitive than the one from the simulation (see image below). 

![grafik](https://github.com/idaholab/moose/assets/113979717/ae21a7fe-5e46-47a5-975e-256782112761)

The only way I could think of making the capillary pressure more sensivity would be, if the curve jump at a certain saturation, so that the capillary pressure could react fast and strong to small changes in saturation at thie specified range. This would not be possible with the relatively simple sigmoidal VG formulation though.

A suggestion for a solution to this problem was made by  Durner (1994, https://doi.org/10.1029/93WR02676), extending the function to account for multimodal poresize distribution, maybe this could be implemented?

 
Cheers

**Comment by GiudGiud** [comment]
@cpgr

**Comment by marinsiebert** [comment]
I did some simple calculations and so far I don't think, that the model by Durner is of much help in this case (although it might still be an interesting addition to the module).

I would still appreciate some input on how to make the model more sensitive to changes in saturation, that is if there is an actual logical on easy approach.
